apis/replication.storage/v1alpha1/volumereplicationclass_webhook.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"errors"
+	"reflect"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// log is for logging in this package.
+var volumereplicationclasslog = logf.Log.WithName("volumereplicationclass-resource")
+
+func (r *VolumeReplicationClass) SetupWebhookWithManager(mgr ctrl.Manager) error {
```

Comment:
Why `r` for `VolumeReplicationClass`? I guess it's used elsewhere and can not easily be modified without changing other files. `r` is not a self-explaining name when using it inside functions.

Reply:
this is also auto generated not manually added, i think it need changes in many places